### PR TITLE
fastai 2.7.18 :snowflake:

### PR DESCRIPTION
**Pull request by msentissi**
fastai 2.7.18 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-6347]
- dev_url:        https://github.com/fastai/fastai/tree/2.7.18
- conda_forge:    https://github.com/conda-forge/fastai-feedstock
- pypi:           https://pypi.org/project/fastai/2.7.18
- pypi inspector: https://inspector.pypi.io/project/fastai/2.7.18

### Explanation of changes:

- upgrade to v2.7.18

[PKG-6347]: https://anaconda.atlassian.net/browse/PKG-6347?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+  sha256: b20593dbcae7522f1d77a8f5163d1fd60314f292640496804dc356e41cb36454
 
 build:
   entry_points:
     - configure_accelerate=fastai.distributed:configure_accelerate
   # torchvision doesn't support s390x or ppc64le
-  skip: true  # [py<37 or s390x or ppc64le]
+  skip: true  # [py<38 or s390x or ppc64le]
+  # fastcore is missing the py312 artifact for win-64
+  skip: true  # [py>=312 and win]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
With https://github.com/AnacondaRecipes/fastcore-feedstock/pull/4

**Review by cbouss — APPROVED** [review]
LGTM, with skip>=3.12 removed.

**Review by skupr-anaconda — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
There are missing artifacts for **py312** on `win64`:
```
conda search fastcore --subdir=win-64
Loading channels: done
# Name                       Version           Build  Channel
fastcore                      1.5.29 py310haa95532_0  repo/main
fastcore                      1.5.29 py310haa95532_1  repo/main
fastcore                      1.5.29 py311haa95532_0  repo/main
fastcore                      1.5.29 py311haa95532_1  repo/main
fastcore                      1.5.29  py38haa95532_0  repo/main
fastcore                      1.5.29  py38haa95532_1  repo/main
fastcore                      1.5.29  py39haa95532_0  repo/main
fastcore                      1.5.29  py39haa95532_1  repo/main
```
Could you build it with the staging channel of https://github.com/AnacondaRecipes/fastcore-feedstock/pull/4?

**Comment by msentissi** [comment]
@skupr-anaconda @cbouss We have the same issue with `fastdownload`, I will rebuild it.

**Comment by msentissi** [comment]
`fastdownload` PR: https://github.com/AnacondaRecipes/fastdownload-feedstock/pull/3

**Pull request merged**